string start_layer
uint8[] input
---
uint8[] output
